cli/src/main/java/org/jboss/galleon/cli/HelpSupport.java

```diff
@@ -168,6 +90,132 @@ private static String pad(int max, String k) {
         return lst;
     }
 
+    public static String buildHelp(CommandRegistry<? extends Command, ? extends CommandInvocation> registry,
+                                   Set<String> commands, boolean footer) throws CommandNotFoundException {
```

Comment:
footer doesn't seem needed in the public exposed method. You could expose a public buildHelp without footer parameter and make this one private.

Reply:
Indeed